### PR TITLE
YLP-2012 Migrate LabeledCheckbox to dumb

### PR DESCRIPTION
**Pull request by celia-rouis**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

**Comment by celia-rouis** [comment]
Related yourloops PR: https://github.com/mdblp/yourloops/pull/835

File: packages/blip/app/components/chart/trendssubnav.js [hunk 1]
```diff
@@ -33,16 +33,13 @@ class DaysGroup extends React.Component {
   }
 
   render() {
-    var groupClass = cx({
-      daysGroup: true,
```
Comment (TimCanu) [inline packages/blip/app/components/chart/trendssubnav.js:37]: Shouldn't `.daysGroup` be deleted from `packages/blip/app/components/chart/trendssubnav.less`?

File: packages/dumb/src/components/controls/labeled-checkbox/labeled-checkbox.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+interface LabeledCheckboxProps {
+  checked: boolean
+  onFn: () => void
+  offFn: () => void
+  label: string
+}
+
+export const LabeledCheckbox: FunctionComponent<LabeledCheckboxProps> = (props) => {
+  const { checked, onFn, offFn, label } = props
```
Comment (TimCanu) [inline packages/dumb/src/components/controls/labeled-checkbox/labeled-checkbox.tsx:39]: What does `onFn` and `offFn` mean?

Comment (celia-rouis) [reply packages/dumb/src/components/controls/labeled-checkbox/labeled-checkbox.tsx:39]: - `onFn`: method called when the checkbox is checked (it's "on")
- `offFn`: method called when the checkbox is unchecked (it's "off")

I thought of some other names but they were very Angular-based (`emitChecked()`, `emitUnchecked()` for instance) but I did not know how to name those in React so I kept the initial names. If you have a better idea, please share it 😄

File: packages/blip/app/components/chart/trendssubnav.js (hunk 1 above)
Comment (celia-rouis) [reply packages/blip/app/components/chart/trendssubnav.js:37]: You're right, good catch 👀

**Review by TimCanu — APPROVED** [review]

**Pull request merged**